dvc/repo/live.py

```diff
@@ -10,15 +15,27 @@
     from dvc.path_info import PathInfo
     from dvc.repo import Repo
 
+webbrowser_open = once_per_args(webbrowser.open)
+
 
 def create_summary(out):
+
     assert out.live and out.live["html"]
 
     metrics, plots = out.repo.live.show(str(out.path_info))
 
     html_path = out.path_info.fspath + "_dvc_plots"
 
-    render(out.repo, plots, metrics=metrics, path=html_path, refresh_seconds=5)
+    index_path = render(
+        out.repo, plots, metrics=metrics, path=html_path, refresh_seconds=5
+    )
+
+    url = index_path.as_uri()
+
+    if "Microsoft" in uname().release:
+        url = Path(index_path) / "index.html"
+
+    webbrowser_open(url)
```

Comment:
Could you please move this piece to `ui`? Maybe as `ui.launch` (similar to [`click.launch`](https://click.palletsprojects.com/en/8.0.x/utils/#launching-applications)) or `ui.open_browser`? We can handle all `relpath`, `uri` and `WSL` stuff inside it. What do you think?

Reply:
Sounds good

Reply:
Opened #6993